gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/LargeMessagingIT.java

```diff
@@ -53,6 +53,16 @@ public void handleUk06WithCidAttachment() throws JSONException {
         verifyBundle("/json/LargeMessage/expectedBundleScenario1.json");
     }
 
+    @Test
+    public void handleUk06WithEMISCidAttachment() throws JSONException {
```

Comment:
`LargeMessaging` has a specific meaning within GP2GP, so I'm a little unhappy that this test lives within `LargeMessagingIT.java`.

Though I think this issue already exists, so perhaps the file needs renaming.